Diplom_3

vadsmirnov opened the pull request:
Диплом_3 Готов

vadsmirnov committed:
Димлом3 Готово

vadsmirnov committed:
добавлен отчет Allure

mkouznetsov-git commented on pom.xml:
```diff
@@ -0,0 +1,161 @@
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind -->
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
```
Дубликат. Такая зависимость уже объявлена выше.

mkouznetsov-git commented on src/test/java/AuthTest.java:
```diff
@@ -0,0 +1,146 @@
+    private String token = "";
+    private LoginUser loginData;
+    private CreateUser createData;
+    private int statusCode;
```
Эти две переменные нигде не используются

mkouznetsov-git commented on src/test/java/AuthTest.java:
```diff
@@ -0,0 +1,146 @@
+    @Test
+    @DisplayName("Авторизация через кнопку на главном экране")
+    public void authMainButtonTest() {
+        //MainPage mainPage = new MainPage(driver);
```
Ненужные комментарии следует удалять из финальной версии кода. Поправь все классы сразу на эту тему.

mkouznetsov-git commented on src/test/java/AuthTest.java:
```diff
@@ -0,0 +1,146 @@
+
+    @After
+    public void tearDown(){
+        ValidatableResponse responseDelete = UserClient.deleteUser(token);
```
Переменная responseDelete не используется

mkouznetsov-git commented on src/test/java/ConstructorTest.java:
```diff
@@ -0,0 +1,73 @@
+    public void checkSause() {
+
+        MainPage mainPage = new MainPage(driver);
+        mainPage.clickButtonSause();
```
Если удалить эту строчку, то тест всё-равно проходит успешно. Значит, проверка перехода на вкладки не работает.

mkouznetsov-git commented on src/test/java/ProfilTest.java:
```diff
@@ -0,0 +1,127 @@
+    private String token = "";
+    private LoginUser loginData;
+    private CreateUser createData;
+    private String email;
```
Переменные email, password, statusCode, isRegistered, shouldBurgerTitle не используются

mkouznetsov-git commented on src/test/java/ProfilTest.java:
```diff
@@ -0,0 +1,127 @@
+
+    @After
+    public void tearDown(){
+        ValidatableResponse responseDelete = UserClient.deleteUser(token);
```
Переменная responseDelete не используется

mkouznetsov-git commented on src/test/java/RegisterTest.java:
```diff
@@ -0,0 +1,102 @@
+import io.qameta.allure.junit4.DisplayName;
+import io.restassured.response.ValidatableResponse;
+import mod.LoginUser;
+import org.apache.commons.lang3.RandomStringUtils;
```
Импорт не используется

mkouznetsov-git commented on src/test/java/RegisterTest.java:
```diff
@@ -0,0 +1,102 @@
+import mod.LoginUser;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.After;
+import org.junit.Assert;
```
Импорт не используется

mkouznetsov-git commented on src/test/java/RegisterTest.java:
```diff
@@ -0,0 +1,102 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import pages.LoginPage;
```
Импорт не используется

mkouznetsov-git commented on src/test/java/RegisterTest.java:
```diff
@@ -0,0 +1,102 @@
+    }
+    @After
+    public void tearDown(){
+        ValidatableResponse responseDelete = UserClient.deleteUser(token);
```
responseDelete не используется

vadsmirnov committed:
Исправлены неиспользуемые перменные.
Исправлен файл pom.xml
Исправлен тестовый класс ConstructorTest, в части проверок перехода по вкладкам
